modules/sdk-core/src/bitgo/lightning/iLightning.ts

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable no-redeclare */
```

Comment:
It looks like this is a pattern in `io-ts` but this may cause accidental problems when other developers update these files and cause duplicate errors. Can we implement another rule for this for the best of both worlds?

Reply:
TypeScript by itself without the eslint rule `no-redeclare` looks to handle it nicely, but we'd have to do a repo wide check for this if we want to verify this approach.

Reply:
what do you mean by "implement another rule for this" ? Should I use `// eslint-disable-next-line no-redeclare` ?